packages/cypher-builder/src/expressions/operations/boolean.ts

```diff
@@ -86,10 +106,43 @@ export function and(...ops: Array<Predicate | undefined>): Predicate | undefined
     return predicate1;
 }
 
+/** Generates an `NOT` operator before the given predicate
+ * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/syntax/operators/#query-operators-boolean)
+ * @group Expressions
+ * @category Operators
+ * @example
+ * ```ts
+ * console.log("Test", Cypher.not(
+ *     Cypher.eq(new Cypher.Literal("Hi"), new Cypher.Literal("Hi"))
+ * );
+ * ```
+ * Translates to
+ * ```cypher
+ * NOT "Hi" = "Hi"
+ * ```
+ *
+ */
 export function not(child: Predicate): BooleanOp {
     return new NotOp(child);
 }
 
+/** Generates an `OR` operator between the given predicates
+ * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/syntax/operators/#query-operators-boolean)
+ * @group Expressions
+ * @category Operators
+ * @example
+ * ```ts
+ * console.log("Test", Cypher.or(
+ *     Cypher.eq(new Cypher.Literal("Hi"), new Cypher.Literal("Hi")),
+ *     new Cypher.Literal(false)).toString()
+ * );
+ * ```
+ * Translates to
+ * ```cypher
```

Comment:
Does this render okay? Sometimes it doesn't if you don't leave whitespaces around code blocks.

Reply:
Good catch, but it renders correctly :)